### DISCUSSION TITLE
Multiple photo at once

### DISCUSSION THREAD

**Discussion by yuretzgt** [eternnoir/pyTelegramBotAPI#1766] [Q&A]
Is it possible to send few photos with text only in one message?

like that

```
file_attach1 = open("file_1.jpg", 'rb').read()
file_attach2 = open("file_2.jpg", 'rb').read()
files = [file_attach1,file_attach2]
bot.send_photo(chat_id,files,caption="Multiple photo at once")
```

**Comment by coder2020official** [comment]
https://pytba.readthedocs.io/en/latest/sync_version/index.html#telebot.TeleBot.send_media_group

**Reply by yuretzgt** [reply]
Thanks, but there is no option to combine it with text

**Reply by yuretzgt** [reply]
TypeError: TeleBot.send_media_group() got an unexpected keyword argument 'caption'

**Comment by Badiboy** [comment]
https://pytba.readthedocs.io/en/latest/types.html#telebot.types.InputMediaPhoto

![image](https://user-images.githubusercontent.com/5613421/198046505-ac2b01fd-842c-4bbc-847a-708c04a68ffd.png)
